tests/Microsoft.Windows.Setting.Accessibility/Microsoft.Windows.Setting.Accessibility.Tests.ps1

```diff
@@ -114,6 +114,35 @@ Describe 'MousePointer' {
     }
 }
 
+Describe 'DynamicScrollbars'{
+   It 'Keeps current value.'{
+      $initialState = Invoke-DscResource -Name AlwaysShowScrollbars -ModuleName Microsoft.Windows.Setting.Accessibility -Method Get -Property @{}
+
+      $parameters = @{ ShowScrollBars = 'KeepCurrentValue' }
+
+      $testResult = Invoke-DscResource -Name AlwaysShowScrollbars -ModuleName Microsoft.Windows.Setting.Accessibility -Method Test -Property $parameters
+      $testResult.InDesiredState | Should -Be $true
+
+      # Invoking set should not change these values.
+      Invoke-DscResource -Name AlwaysShowScrollbars -ModuleName Microsoft.Windows.Setting.Accessibility -Method Set -Property $parameters
+      $finalState = Invoke-DscResource -Name AlwaysShowScrollbars -ModuleName Microsoft.Windows.Setting.Accessibility -Method Get -Property @{}
+      $finalState.ShowScrollBars | Should -Be $initialState.ShowScrollBars
+   }
```

Comment:
No longer needed

Reply:
Why not? Why do the other resources need the section but this one does not? Why the sudden lack of concern about retaining this value - are the other settings "special" in some way?

Reply:
This setting only has two states, on or off. The other settings can have variable input like MousePointer size and magnification. A user may have a custom scale factor that they have set and we want to make sure we don't overwrite those values if it is different from the default set. 

This is basically a switch parameter so you only need to know if it's true or false.